bin/kibana

```diff
@@ -21,4 +21,7 @@ if [ ! -x "$NODE" ]; then
   exit 1
 fi
 
-NODE_ENV=production exec "${NODE}" --no-warnings --max-http-header-size=65536 $NODE_OPTIONS "${DIR}/src/cli" ${@}
+BASE_NODE_OPTIONS="--no-warnings --max-http-header-size=65536"
+PARSED_NODE_OPTIONS="${NODE_OPTIONS:+" $NODE_OPTIONS"}"
+
+NODE_OPTIONS="${BASE_NODE_OPTIONS}${PARSED_NODE_OPTIONS}" NODE_ENV=production exec "${NODE}" "${DIR}/src/cli" ${@}
```

Comment:
Any reason not to keep it how it was but re-export NODE_OPTIONS without needing the additional variables?

`NODE_OPTIONS="--no-warnings --max-http-header-size=65536 ${NODE_OPTIONS}" NODE_ENV=production exec "${NODE}" "${DIR}/src/cli" ${@}`

Reply:
one plus is we don't have duplicates, in practice it hasn't been an issue but it could result in two max-http-header-size settings.  the second one takes precedence

Reply:
@tylersmalley @jbudz I like the idea of re-exporting node_options. I just updated the entire PR